base_suspend_security/tests/test_base_suspend_security.py

```diff
@@ -46,3 +56,20 @@ def test_suspend_security_on_search(self):
             model.sudo(user_without_access).search([])
         # this tests the search
             model.sudo(user_without_access).suspend_security().search([])
```

Comment:
Isn't this line indented "too much"?

```suggestion
        model.sudo(user_without_access).suspend_security().search([])
```

Reply:
thanks, I always strife for minimal diffs, but fixing this on the way doesn't hurt